setup.cfg

```diff
@@ -20,10 +20,13 @@ package_dir =
 packages = find:
 python_requires = >=3.6
 install_requires =
-    vitessce==1.0.5
+    vitessce==1.0.8
     hubmap-commons>=2.0.12
     requests>=2.27.1
     nbformat==5.1.3
+    zarr>=2.11.1
+    aiohttp>=3.8.1
+    fsspec>=2022.3.0
```

Comment:
So we've got this weird situation where we're trying to make sure this code can run under Python 3.6 (... not a situation I'm happy about!) `aiohttp` is fine, but it looks like the other two won't work in 3.6:

```
$ python --version
Python 3.6.13 :: Anaconda, Inc.

$ pip install 'zarr>=2.11.1'
ERROR: Could not find a version that satisfies the requirement zarr>=2.11.1 (from versions: 0.2.2, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.3.0, 0.4.0, 1.0.0b3, 1.0.0b4, 1.0.0b6, 1.0.0, 1.1.0, 2.0.0a2, 2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.2.0rc1, 2.2.0rc2, 2.2.0rc3, 2.2.0, 2.3.0, 2.3.1, 2.3.2, 2.4.0, 2.5.0, 2.6.1, 2.7.0, 2.7.1, 2.8.0, 2.8.1, 2.8.2, 2.8.3)
ERROR: No matching distribution found for zarr>=2.11.1

ERROR: Could not find a version that satisfies the requirement fsspec>=2022.3.0 (from versions: 0.1.0, 0.1.1, 0.1.2, 0.1.3, 0.1.4, 0.2.0, 0.2.1, 0.2.2, 0.2.3, 0.3.0, 0.3.1, 0.3.2, 0.3.3, 0.3.4, 0.3.5, 0.3.6, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.4.4, 0.4.5, 0.5.1, 0.5.2, 0.6.0, 0.6.1, 0.6.2, 0.6.3, 0.7.0, 0.7.1, 0.7.2, 0.7.3, 0.7.4, 0.8.0, 0.8.1, 0.8.2, 0.8.3, 0.8.4, 0.8.5, 0.8.6, 0.8.7, 0.9.0, 2021.4.0, 2021.5.0, 2021.6.0, 2021.6.1, 2021.7.0, 2021.8.1, 2021.9.0, 2021.10.0, 2021.10.1, 2021.11.0, 2021.11.1, 2022.1.0)
ERROR: No matching distribution found for fsspec>=2022.3.0
```

If by any chance you could work with an older version for both of these, great! Otherwise, I don't think 3.6 problems should block this work, but I should tell Nils that the other track will not be moving forward. (The big picture there is that we want to determine at index time if there if there is a visualization, and PSC is stuck in a 3.6 environment for some reason.)

So, nothing actionable for you, but I'll link to this comment from the UI agenda to explain how things stand. Sorry to be long-winded!